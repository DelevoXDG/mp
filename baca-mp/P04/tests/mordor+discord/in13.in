1
21
New T1 1q
DelLast T1 T2
DelFirst T2 T1
Display T1
Display T2
Display T3
Display T4
InsertFirst T1 w2
Display T1
Display T2
Display T3
Display T4
Trains
DelLast T1 T3
Trains
DelFirst T1 T4
Trains
Display T1
Display T2
Display T3
Display T4