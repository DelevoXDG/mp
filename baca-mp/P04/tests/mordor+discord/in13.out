T1: 1q
Train T2 does not exist
Train T3 does not exist
Train T4 does not exist
T1: w2 1q
Train T2 does not exist
Train T3 does not exist
Train T4 does not exist
Trains: T1
Trains: T3 T1
Trains: T4 T3
Train T1 does not exist
Train T2 does not exist
T3: 1q
T4: w2